2018/hsg-vernetzung.md

```diff
@@ -0,0 +1,110 @@
+Title: Vernetzung von Hochschulgruppen mit FOSS-Bezug
+Date: 2018-10-29 20:00
+Category: Artikel
+Tags: Freie Software, Gesellschaft, Hochschule, Vernetzung
+Slug: hsg-vernetzung
+Author: Carsten
+Summary: Hochschulen sind ein guter Ansatzpunkt, um sich für mehr und bessere Freie Software (FOSS) in der Gesellschaft einzusetzen. Dazu braucht es lokale Akteur:innen, die gegenseitig vernetzt sind, um an den großen Rädern zu drehen. Zu dieser Vernetzung möchten wir beitragen.
+
+Vor ca. vier Jahren, im Dezember 2014, wurde die Hochschulgruppe für Freie Software und Freies Wissen Dresden gegründet. Anlass war die Unzufriedenheit einiger Leute (Studierende und Mitarbeiter) über die Dominanz von proprietärer Software im digitalen Alltag an den beiden Hochschulen TU und HTW.
+Die Motivation die Organisationsform einer Hochschulgruppe zu wählen waren folgende:
+
+* Hochschulen haben eine überproportional hohe Dichte an interessierten und somit potentiell engagierten Leuten.
+* Quasi alle zukünftigen Entscheidungsträger:innen durchlaufen das Hochschulsystem und sollten über FOSS Bescheid wissen.
+* Hochschulgruppen erhalten einfachen Zugang zu Veranstaltungsräumen und Kommunikationskanälen (Aushänge, Rundmails).
```

Comment:
+ * Im Gegensatz zu weitreichend bekannten NGOs ist eine Hochschulgruppe in der Organisation integriert und wird daher eventuell weniger mißtrauisch in der Institution wahrgenommen.

Reply:
Stimme ich zu, würde ich aber nicht so an die große Glocke hängen...